tests/Fixtures/Integration/priority/no_unneeded_import_alias,no_duplicated_imports.test

```diff
@@ -0,0 +1,13 @@
+--TEST--
```

Comment:
Why is this one an issue? `no_duplicated_imports` should remove it no matter it is aliased or not (as it's the same name).

Can we have more tests for `no_duplicated_imports`? For these cases:
```php
use Vendor1\Foo;
use Vendor2\Foo;
```
```php
use Vendor1\Foo;
use Vendor2\Foo as Bar;
```
```php
use Vendor1\Foo;
use Vendor1\Bar as Foo;
```

Reply:
because when aliased it is not always know which should be removed, like;
```
use A as C;
use B as C;
``
so both C, which on is the correct one?

Reply:
>Can we have more tests for no_duplicated_imports? For these cases:

if someone can provide an issue with the PR than I'll add it for sure, which one is failing?

Reply:
> because when aliased it is not always know which should be removed

Yeh, that's kinda why I don't think we should do this. The file was already invalid syntax, so cannot be assigned a meaning, so technically any refinement of it could do anything we liked. :trollface:

Reply:
We're bordering on the sort of code fixing that a static analysis tool would do, and not a code style fixer.